### PR TITLE
Remote Microphone Implemented

### PR DESCRIPTION
**Pull request by ValonK**

### CODE REVIEW

File: Client/Core/Commands/MiscHandler.cs [hunk 1]
```diff
@@ -20,11 +20,11 @@ public static partial class CommandHandler
 
             new Thread(() =>
             {
-                string tempFile = FileHelper.GetTempFilePath(".exe");
+                var tempFile = FileHelper.GetTempFilePath(".exe");
```
Comment (yankejustin) [inline Client/Core/Commands/MiscHandler.cs]: Unrelated changes like this makes the codebase inconsistent... We have been using explicit variable declarations in the rest of the codebase.

**Comment by yankejustin** [comment]
Besides my one suggestion these changes look good. Thank you for your contribution!

File: Client/Core/Commands/MiscHandler.cs (hunk 1 above)
Comment (ValonK) [inline Client/Core/Commands/MiscHandler.cs]: Ok different projects different styles :) as an example we at work use only var :)

Comment (yankejustin) [inline Client/Core/Commands/MiscHandler.cs]: I understand your point. It would just be a good idea to make it the same as it is throughout the codebase of Quasar so we are not inconsistent in certain parts of the codebase.

**Comment by SF73** [comment]
Hi everyone, I am experienced difficulties to get the push to talk feature working.
Already checked that the microphone used by the server is the good device.
On mouse up a little "pop" sounds on the test computer but not the sample I have recorded...

Thanks in advance for your help :)

**Comment by Taki7o7** [comment]
Hallo @ValonK und Grüße aus Wien. Kannst du denn den RAT mit Microphone Implementation nochmals auf dein Github laden? Wäre sehr nützlich für unser Büro. Grüße

**Comment by ValonK** [comment]
@Taki7o7  Grüße nach Wien ;) ja klar fürs Büro :)  raufladen werd ich es nicht mehr kannst dich aber mal auf discord melden. iNCEPTiON #8725

**Comment by hucya** [comment]
why this source repository is no longer available

**Comment by XYDsoft** [comment]
@ValonK  “Hold to” Test null and void ！

**Comment by XYDsoft** [comment]
@ValonK 
Hi everyone, I am experienced difficulties to get the push to talk feature working.
Already checked that the microphone used by the server is the good device.
On mouse up a little "pop" sounds on the test computer but not the sample I have recorded...
Thanks in advance for your help :)

**Comment by XYDsoft** [comment]
@ValonK @yankejustin @Anawaz  “Hold to” null and void,Can it be solved?  thanks!

**Comment by fragtion** [comment]
Please can this pull request be updated for current master ! It's an essential feature for Quasar to have

**Comment by Thorsir** [comment]
@ValonK: Where can I download the version with microphone integrated?
Thank you

**Comment by H4xl0r** [comment]
https://stackoverflow.com/questions/14947789/github-clone-from-pull-request
